swagger.yaml

```
@@ -4929,6 +5181,39 @@ components:
       example:
         - label: "1968"
           cbor_metadata: "\\xa100a16b436f6d62696e6174696f6e8601010101010c"
+    tx_content_redeemers:
+      type: array
+      items:
+        type: object
+        properties:
+          tx_index:
```

```suggestion
          cert_index:
```
Shouldn't this be `cert_index`?